test/unit/abci/handlers/endBlockHandlerFactory.spec.js

```diff
@@ -273,4 +292,47 @@ describe('endBlockHandlerFactory', () => {
 
     expect(response.validatorSetUpdate).to.be.equal(validatorSetUpdate);
   });
+
+  it('should return consensusParamUpdates if request contains update consensus features flag', async function it() {
+    const getLatestFeatureFlagGetMock = this.sinon.stub();
+    getLatestFeatureFlagGetMock.withArgs('block').returns({
+      maxBytes: 1,
+      maxGas: 2,
+    });
+    getLatestFeatureFlagGetMock.withArgs('evidence').returns({
+      maxAgeNumBlocks: 1,
+      maxAgeDuration: null,
+      maxBytes: 2,
+    });
+    getLatestFeatureFlagGetMock.withArgs('version').returns({
+      appVersion: 1,
+    });
+
+    getLatestFeatureFlagMock.resolves({
+      get: getLatestFeatureFlagGetMock,
+    });
+
+    const response = await endBlockHandler(requestMock);
+
```

Comment:
Let’s verify proto object to make sure that all fields are properly set or toJSON already return only defined onces?